File: src/net/eduard/api/command/api/ApiCommand.java

package net.eduard.api.command.api;

import net.eduard.api.lib.manager.CommandManager;

public class ApiCommand extends CommandManager {

	public ApiCommand() {
		super("api");
		register(new ApiHelpCommand());
		register(new ApiReloadCommand());
		register(new ApiUnloadWorldCommand());
		register(new ApiLoadWorldCommand());
		register(new ApiWorldsCommand());
		register(new ApiDeleteWorldCommand());
		register(new ApiListCommand());
		register(new ApiDisableCommand());
		register(new ApiEnableCommand());
		register(new ApiListCommand());
		register(new ApiReloadConfigCommand());
		register(new ApiSaveConfigCommand());
		register(new ApiSaveCommand());
		register(new ApiReloadCommand());
		register(new ApiUnloadCommand());

	}

}

File: src/main/java/net/eduard/api/command/api/ApiCommand.kt
package net.eduard.api.command.api

import net.eduard.api.lib.manager.CommandManager

class ApiCommand : CommandManager("api") {
    init {
        register(ApiHelpCommand())
        register(ApiSQLDebugCommand())
        register(ApiReloadCommand())
        register(ApiUnloadWorldCommand())
        register(ApiLoadWorldCommand())
        register(ApiWorldsCommand())
        register(ApiDeleteWorldCommand())
        register(ApiListPluginsCommand())
        register(ApiDisablePluginCommand())
        register(ApiEnablePluginCommand())
        register(ApiRestartEduardAPICommand())
        register(ApiListPluginsCommand())
        register(ApiSaveCommand())
        register(ApiReloadCommand())
        register(ApiUnloadPluginCommand())
        register(ApiLoadPluginCommand())
    }
}